demos/README.md

```diff
@@ -43,6 +43,7 @@ The Open Model Zoo includes the following demos:
 - [Pedestrian Tracker C++ Demo](./pedestrian_tracker_demo/cpp/README.md) - Demo application for pedestrian tracking scenario.
 - [Security Barrier Camera C++ Demo](./security_barrier_camera_demo/cpp/README.md) - Vehicle Detection followed by the Vehicle Attributes and License-Plate Recognition, supports images/video and camera inputs.
 - [Speech Recognition Python\* Demo](./speech_recognition_demo/python/README.md) - Speech recognition demo: takes audio file with an English phrase on input, and converts it into text.
+- [Speech Recognition Offline Python\* Demo](./speech_recognition_offline_demo/python/README.md) - Offline speech recognition demo: takes a whole audio file with an English phrase on input, and converts it into text.
```

Comment:
IMO, this doesn't sufficiently explain the difference between online and offline. Maybe the description of the online demo should be amended to clarify that it does the conversion chunk-by-chunk.

Reply:
Currently `speech_recognition_demo` processes a whole file. Your statement will become true after https://github.com/openvinotoolkit/open_model_zoo/pull/2108 merge. I think it should be rephrased along with `speech_recognition_demo` renaming and be done in https://github.com/openvinotoolkit/open_model_zoo/pull/2108.

Reply:
Okay, fair enough.